README.md

```diff
@@ -336,6 +330,22 @@ If not already in the docker container, run `docker exec -it keycloak bash`
 
 Once you have identified the line that you'd like to edit, copy and paste it into the messages `messages_en.properties`
 
+### Troubleshoot for Running Locally
+
+#### If local_setup.sh script fails to run
+
+1. Copy the [tdr-realm-export.json](https://github.com/nationalarchives/tdr-configurations/blob/master/keycloak/tdr-realm-export.json) from the tdr-configurations repository into the tdr-auth-server directory
+2. Navigate to the cloned repository: `$ cd tdr-auth-server`
+3. Build the TDR theme:
+    * If npm is not installed install [nvm](https://github.com/nvm-sh/nvm#intro) in root directory
+    * Once nvm is installed run: `[root directory] $ nvm install 16.17.0`
+    * Run the following commands in the root directory:  `[root directory] $ npm install` and `[root directory] $ npm run build-theme`
+        * this will compile the theme sass, copy the static assets to the theme `resource` directory and compile the typescript for WebAuthn.
+4. Build all three spi jars:
+    * From the root directory run the following command: `sbt assemblyPackageDependency assembly`
+    * This will generate the jar for the GovUK Notify service and the Event Publisher service
```

Comment:
This was out of date before your PR but there's now a credential provider service as well so could you change this to "GovUK Notify service, the Event Publisher service and the credential provider service" please. This is copied as a comment in the script below as well so could you change it there too please.